RELEASES.md

```diff
@@ -27,10 +45,10 @@ ink! 3.0-rc4 is compatible with
     - Install the newest version using `cargo install --force cargo-contract`.
 - Substrate version `4.0.0-dev` including the `contracts-pallet` version `4.0.0-dev`.
 - [`substrate-contracts-node`](https://github.com/paritytech/substrate-contracts-node) version `0.1.0` or newer.
-    - Install the newest version using `cargo install substrate-contracts-node --git https://github.com/paritytech/substrate-contracts-node.git --force`.
+    - Install the newest version using `cargo install contracts-node --git https://github.com/paritytech/substrate-contracts-node.git --force`.
```

Comment:
The naming is a bit unfortunate. The crate is called `contracts-node`, but the executable and the project are called `substrate-contracts-node`. Don't want to change it right now though, I think we've made enough breaking changes this week.

Reply:
Yeah that is slightly annoying